publisher-sdk/src/main/java/com/criteo/publisher/logging/SdkLogMessage.kt

```diff
@@ -0,0 +1,38 @@
+/*
+ *    Copyright 2020 Criteo
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package com.criteo.publisher.logging
+
+import android.util.Log
+import com.criteo.publisher.model.AdUnit
+
+internal object SdkLogMessage {
+
+  @JvmStatic
+  fun sdkInitialized(
```

Comment:
From a naming perspective, I think `createSdkInitializedLog` would be more explicit (current naming makes it feel like it's an event).

Reply:
This is general to all `LogMessage`:
What do you think about using a `on` prefix to show that it is not an action.
For instance:

```
# declaration
internal object DummyMessage {
  fun onSdkInitialized() = /* ... */

  fun onMoPubConsentSet() = /* ... */

  fun onAdObjectEnrichedWithNoBid() = /* ... */
}

# caller
logger.log(DummyMessage.onSdkInitialized())
logger.log(DummyMessage.onMoPubConsentSet())
logger.log(DummyMessage.onAdObjectEnrichedWithNoBid())

# or caller with static import
logger.log(onSdkInitialized())
logger.log(onMoPubConsentSet())
logger.log(onAdObjectEnrichedWithNoBid())
```

I'm afraid that the `create` prefix and `Log` suffix is redondant with the context at call-site or at declaration-site.

Reply:
Ok sounds good to me.